### PR TITLE
Update reflectometry release notes

### PR DESCRIPTION
**Pull request by gemmaguest**
Update the release notes for reflectometry:
- make text more concise and readable
- reorganise notes with more interesting items first
- add lots of screenshots for the GUI changes

**To test:**

Proof read and check that layout, screenshots etc. are ok

Refs #26232

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/release/v4.1.0/reflectometry.rst [hunk 1]
```diff
@@ -2,84 +2,160 @@
...
-- The first and second transmission runs are now entered via two separate boxes. Multiple runs can be summed for either of these inputs by entering the run numbers as a comma-separated list.
-- New inputs have been added to control stitching of transmission runs: you can select whether to scale the left or right workspace; you can enter rebin parameters specifically for transmission run stitching (previously the same parameters as stitching of the output workspaces were used).
-- Spectra of interest can be specified for the transmission runs via a new column on the Experiment Settings tab (previously the same spectra as the run workspaces was used).
-- Default values for the current instrument are automatically set in the Experiment and Instrument Settings tabs when the interface is opened or the instrument is changed.
-- Processing in event mode is now done asynchronously. Previously this used to lock up MantidPlot.
-- Error handling has been improved to catch invalid inputs earlier. Errors are highlighted in red or (for the table) with a red star.
-- The progress bar now remembers previous progress when you pause and restart processing.
+The Toobar
```
Comment (Pasarus) [inline docs/source/release/v4.1.0/reflectometry.rst]: I have a feeling this means Toolbar?
```suggestion
The Toolbar
```

File: docs/source/release/v4.1.0/reflectometry.rst [hunk 2]
```diff
@@ -2,84 +2,160 @@
...
+.. figure:: ../../images/ISISReflectometryInterface_table_editing.png
+  :class: screenshot
+  :align: right
+  :alt: Editing the Runs table
+
+  *Editing the Runs table*
+
+Navigation by keyboard shortcuts has been added to the Runs table:
+
+- ``F2`` edits a cell
```
Comment (Pasarus) [inline docs/source/release/v4.1.0/reflectometry.rst]: Perhaps add a table with all of the keyboard shortcuts somewhere?

**Review by Pasarus — COMMENTED** [review]
Just one thing and a suggestion.

**Review by NickDraper — DISMISSED** [review]
good to go

**Pull request merged**